fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpIdentifierConverter.java

```diff
@@ -80,12 +78,13 @@ public HttpIdentifierConverter(final UriBuilder uriBuilder) {
         this.uriTemplate = new UriTemplate(uriBuilder.toTemplate());
     }
 
-    private UriBuilder uriBuilder() {
-        return UriBuilder.fromUri(uriBuilder.toTemplate());
-    }
-
-    @Override
-    protected String doForward(final String httpUri) {
+    /**
+     * Convert an external URI to an internal ID.
+     *
+     * @param httpUri the external URI.
+     * @return the internal identifier.
+     */
+    protected String toInternalId(final String httpUri) {
```

Comment:
I think this has to be public, otherwise you won't be able to call it from outside the `http/api/rdf` package. Same with `toExternalId`